src/prelude.rs

```diff
@@ -31,4 +31,5 @@ pub use crate::{
     symbols::{self, Marker},
     terminal::{CompletedFrame, Frame, Terminal, TerminalOptions, Viewport},
     text::{self, Line, Masked, Span, Text},
+    widgets::{block::BlockExt, StatefulWidget, Widget},
```

Comment:
I think it makes sense to add StatefulWidget and Widget to the prelude. Wvery app that uses Ratatui calls render on one or more widgets, and these are not likely to cause problems with existing code.